velox/substrait/TypeUtils.cpp

```diff
@@ -43,6 +43,31 @@ int64_t bytesOfType(const TypePtr& type) {
   }
 }
 
+void getTypesFromCompoundName(
+    const std::string& compoundName,
+    std::vector<std::string>& types) {
+  types.clear();
+  // Get the position of ":" in the compoundName.
+  std::size_t pos = compoundName.find(":");
+  // Get the parameter types.
+  std::string funcTypes;
```

Comment:
nit: move this variable next to first use and combine declaration with initialization

```
auto funcTypes = compoundName.substr(pos + 1);
```

Reply:
> compoundName.substr(

its fixed.